edx/analytics/tasks/database_imports.py

```diff
@@ -510,6 +510,27 @@ def columns(self):
         ]
 
 
+class ImportShoppingCartInvoiceTransactions(ImportMysqlToHiveTableTask):
+    """
+    Imports invoice transaction data from an external LMS DB shopping cart table to both a
+    destination directory and a HIVE metastore.
+    """
+    @property
+    def table_name(self):
+        return 'shoppingcart_invoicetransaction'
+
+    @property
+    def columns(self):
+        return [
+            ('id', 'INT'),
+            ('created', 'TIMESTAMP'),
+            ('amount', 'DECIMAL'),
+            ('currency', 'STRING'),
+            ('status', 'STRING'),
+            ('invoice_id', 'INT'),
+        ]
+
```

Comment:
This table has the following columns on my installation:
```
mysql> describe shoppingcart_invoicetransaction;
+---------------------+---------------+------+-----+---------+----------------+
| Field               | Type          | Null | Key | Default | Extra          |
+---------------------+---------------+------+-----+---------+----------------+
| id                  | int(11)       | NO   | PRI | NULL    | auto_increment |
| created             | datetime(6)   | NO   |     | NULL    |                |
| modified            | datetime(6)   | NO   |     | NULL    |                |
| amount              | decimal(30,2) | NO   |     | NULL    |                |
| currency            | varchar(8)    | NO   |     | NULL    |                |
| comments            | longtext      | YES  |     | NULL    |                |
| status              | varchar(32)   | NO   |     | NULL    |                |
| created_by_id       | int(11)       | NO   | MUL | NULL    |                |
| invoice_id          | int(11)       | NO   | MUL | NULL    |                |
| last_modified_by_id | int(11)       | NO   | MUL | NULL    |                |
+---------------------+---------------+------+-----+---------+----------------+
10 rows in set (0.00 sec)
```
Did you include only a subset of these columns intentionally?  The other import jobs import all available columns even when they are not needed further down the line.